### PR TITLE
Milestone 1 - build the project foundation

### PR DESCRIPTION
**Pull request by tGodson**
Initialise project with create-react-app

### CODE REVIEW

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
You have done a good job installing all the required packages and properly setting up the project. 
Let's however, make one minor change before merging.
- [x] add a descriptive title to this PR. This makes the reviewer know what the PR is about at a glance.


### Martin Afani

**Review by christianotieno — APPROVED** [review]
### Milestone Approved 🍾 
Hello @tGodson  👋 
 ##
This milestone has met all the requirements needed for approval. 👏 
Congratulations on the hard work done on this milestone. Good job 👍 
I will go ahead and mark this as complete. Good luck with the next tasks 🥇 
##
### Happy coding 💻 
#### Christian Otieno (TSE)

**Pull request closed without merging**
